boards/remote-reva/README.md

```diff
@@ -105,7 +105,7 @@ More Reading
 2. [CC2538 System-on-Chip Solution for 2.4-GHz IEEE 802.15.4 and ZigBee applications (SWRU319B)][cc2538]
 3. [CC1200 sub-1GHz RF transceiver][cc1200]
 
-[remote-site]: http://www.zolertia.io/products "Zolertia RE-Mote"
+[remote-site]: https://zolertia.io/store/ "Zolertia RE-Mote"
```

Comment:
nit-picky: that's the on-line store front page, not even the Re-Mote specific sub-page as suggested by the link title

Reply:
They don't sell the rev a or pa any more so I thought it would be weird to point to a different board bit I will update to the only purchasable board.